### DISCUSSION TITLE
Thank you.

### DISCUSSION THREAD

**Discussion by palmmc** [UltimatChamp/CustomItemTextures#5] [Show and tell]
### Request Description

I just wanted to say thank you.
It's one thing to port a mod over to the newest version, which is already great. It's another thing to take on the responsibility of updating a mod "more reliably". It's all I've ever wanted for this mod; it's always been crazy to me that mods like Sodium and Iris are immediately updated, but CIT (which is the foundation of so many packs) just doesn't get ported half the time.

So thank you. Have a great day.

**Comment by UltimatChamp** [comment]
❤️
